docs/json/sonarr/cf/anime-lq-groups.json

```diff
@@ -636,6 +636,15 @@
         "value": "\\b(MinisCuba)\\b"
       }
     },
+    {
+      "name": "MiniMTBB",
```

Comment:
sorting is off, should be before `MinisCuba`

Reply:
Fixed